rotkehlchen/chain/ethereum/transactions.py

```diff
@@ -27,6 +27,8 @@
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
+LOGS_QUERY_BLOCKS_RANGE: Final = 900_000
```

Comment:
We already have a range.Check EVM inquirer. For web3 nodes we have `WEB3_LOGQUERY_BLOCK_RANGE = 250000` and for etherscan `300000`

Internally the get_log function splits it.

Would it not make more sense to change the get_logs function and add an optional callback to do in between log queries? And then move the cache writing code into that callback?